README.md

```diff
@@ -337,7 +337,10 @@ $ PORT=8080 parse-server --appId APPLICATION_ID --masterKey MASTER_KEY
 For the full list of configurable environment variables, run `parse-server --help`.
 
 ### Available Adapters
-[Parse Server Modules (Adapters)](https://github.com/parse-server-modules)
+
+All official adapters are distributed as scoped pacakges on [npm (@parse)](https://www.npmjs.com/search?q=scope%3Aparse).
```

Comment:
Cool! Should we keep the github modules as well or are we deprecating that altogether in favor of npm?

Reply:
all official adapters have moved from parse-server-modules to parse-community. Others are still there but not migrated yet.